### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by johnbindel**
Added "Async" suffix on async methods
Minor grouping/ordering adjustments

### CODE REVIEW

**Comment by johnbindel** [comment]
This contains a breaking naming change to IActorManager's GetActorsFromStateProvider method. I renamed it to GetActorsFromStateProviderAsync to follow best practices around async method naming. If necessary, I can resubmit without that change.

**Comment by amanbha** [comment]
Yes, lets not break public api surface area. Please resubmit with changes limited to internal and private methods only.

**Comment by johnbindel** [comment]
Ok, I reverted that commit and addressed only private and internal methods.

File: src/Microsoft.ServiceFabric.Services/Client/ServicePartitionResolver.cs [hunk 1]
```diff
@@ -506,7 +506,7 @@ public static ServicePartitionResolver GetDefault()
                 cancellationToken);
         }
 
-        private async Task<ResolvedServicePartition> ResolveAsyncHelper(
+        private async Task<ResolvedServicePartition> ResolveAsyncHelperAsync(
```
Comment (amanbha) [inline src/Microsoft.ServiceFabric.Services/Client/ServicePartitionResolver.cs]: rename to ResolveHelperAsync?

Comment (johnbindel) [inline src/Microsoft.ServiceFabric.Services/Client/ServicePartitionResolver.cs]: Done

**Pull request merged**